lib/coverband/adapters/hash_redis_store.rb

```diff
@@ -25,14 +25,14 @@ def initialize(redis, opts = {})
         @save_report_batch_size = opts[:save_report_batch_size] || 100
         @format_version = REDIS_STORAGE_FORMAT_VERSION
         @redis = redis
-        raise "HashRedisStore requires redis >= 2.6.0" unless supported?
+        raise "HashRedisStore requires redis >= 6.0.0" unless supported?
```

Comment:
Do we have a reason to require redis 6, we were checking this based on features that redis supports but we aren't using any of the more modern and newer redis features.